soil-service/app/repositories/fields_repository.rb

```diff
@@ -0,0 +1,15 @@
+class FieldsRepository
```

Comment:
If you add new classes it would good to cover them with unit specs :slightly_smiling_face: